app/views/children/show.html.erb

```diff
@@ -53,12 +53,70 @@
               <dt>Comments:</dt>
               <dd><%= @child.comments %></dd>
             </dl>
+            <% unless Flipper[:family_requests].enabled?(current_partner) %>
+              <%= link_to 'Edit Child Details', edit_child_path(@child), class: "btn btn-success" %>
+              <%= link_to 'All Children', children_path, class: "btn btn-info" %>
+            <% end %>
+          </div>
 
+          <% if Flipper[:family_requests].enabled?(current_partner) %>
             <%= link_to 'Edit Child Details', edit_child_path(@child), class: "btn btn-success" %>
-            <%= link_to 'All Children', children_path, class: "btn btn-info pull-right" %>
-          </div>
+            </br>
+            <%= link_to 'All Children', children_path, class: "btn btn-info" %>
+          <% end %>
         </div>
       </div>
+
+      <% if Flipper[:family_requests].enabled?(current_partner) %>
+        <div class="col-9">
+          <div class="card mb-6">
+            <h5 class="card-header bg-primary text-white">Requested Items</h5>
+            <div class="card-body">
+              <% if @child.child_item_requests.count > 0 %>
+                <table class="table table-striped">
+                  <thead>
+                    <tr>
+                      <th scope="col">Request Date</th>
+                      <th scope="col">Item Ordered</th>
+                      <th scope="col">Item Picked Up</th>
+                      <th scope="col">Quantity Ordered</th>
+                      <th scope="col">Quantity Picked Up</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    <% @child_item_requests.each do |child_item_request| %>
+                      <tr>
+                        <td><b><%= child_item_request.created_at.to_date.iso8601 %></b></th>
+                        <td><%= valid_items.dig(child_item_request.ordered_item_diaperid, 'name').to_s %></th>
+                        <td><b>
+                          <% if (diaperid = child_item_request.picked_up_item_diaperid) && child_item_request.picked_up? %>
```

Comment:
We should really extract some of this logic at some point in the future. Might be a good use of actionview-component in the near future.